index.js

```diff
@@ -289,9 +289,9 @@ async function main (argv) {
     process.stderr.write(`\nUnable to write socket.ini: ${err.stack ?? err.message}\n`)
     process.exit(1)
   }
-  process.stdout.write('Ok.')
+  process.stdout.write('Ok.\n')
 
-  process.stdout.write('\nCopying project boilerplate...')
+  process.stdout.write('Copying project boilerplate...')
```

Comment:
Just checking, this one doesn't need a trailing `\n`?

Reply:
This doesn't need one. 
``` 
Copying project boilerplate...Ok.   

Type 'npm start' to launch the app
PS C:\socketsupply\create-socket-app-react-ts> 
```

Reply:
P.S Unless we want to have Ok. in new lines?

This is a final look of the process of initialization. 
```
Initializing npm package...Ok.
Installing developer dependencies...Ok.
Installing '@socketsupply/socket' locally (ssc not in PATH)
Installing dependencies...
running pre req script at node_modules\@socketsupply\socket-win32-x64\bin\install-pre-reqs.js
Using toolchain: vsbuild
# C:\WINDOWS\System32\vcruntime140_1.dll found.
Git found at, changing path to: git.exe
# Found git.exe
Calling vcvars64.bat
Please run in future dev sessions: 
(Or just run cd C:\socketsupply\create-socket-app-react-ts\node_modules\@socketsupply\socket-win32-x64\ps1; .\bin\install.ps1)
$env:PATH='C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\Llvm\x64\bin'+$env:PATH
Ok.
Adding package scripts...Ok.
Creating socket files...Ok. 
Creating tsconfig...Ok.
Setting up TS configuration...Ok.   
Updating project configuration...Ok.
Copying project boilerplate...Ok.   

Type 'npm start' to launch the app
```

Reply:
@wujekbizon I understand now, thanks!